bin/midtrans

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env sh
-
-veritrans $@
```

Comment:
btw since bin files are deleted, when installing veritrans gem there is `warning` message

<img width="718" alt="Screen Shot 2021-09-30 at 10 58 03" src="https://user-images.githubusercontent.com/13027142/135384949-92f0c3ed-cbfd-4c3a-852b-2182a6a305a0.png">

it can be fixed by removing this line: https://github.com/veritrans/veritrans-ruby/blob/809ea9869075f8de7a93d0ed82cdce6ee2dd122f/veritrans.gemspec#L18